tests/test_picking.py

```diff
@@ -146,6 +185,44 @@ def test_horizon_picking():
     # pick nothing
     picker.Pick(0, 0, 0, renderer)
     # 'c' to clear
-    clear_callback = plotter._key_press_event_callbacks['c']
+    clear_callback = plotter.iren._key_press_event_callbacks['c']
     clear_callback[0]()
     plotter.close()
+
+
+def test_enable_fly_to_right_click(sphere):
+
+    point = []
+    def callback(click_point):
+        point.append(click_point)
+
+    pl = pyvista.Plotter()
+    pl.add_mesh(sphere)
+    pl.enable_fly_to_right_click(callback=callback)
+    pl.show(auto_close=False)
+    width, height = pl.window_size
+    cpos_before = pl.camera_position
+    pl.iren._mouse_right_button_press(width//2, height//2)
+
+    # ensure callback was called and camera position changes due to "fly"
+    assert cpos_before != pl.camera_position
+    assert point
+
+
+# def test_enable_fly_to_right_click_multi_render(sphere):
+#     """Same as enable as fly_to_right_click except with two renders for coverage"""
+#     point = []
+#     def callback(click_point):
+#         point.append(click_point)
+
+#     pl = pyvista.Plotter(shape=(1, 2))
+#     pl.add_mesh(sphere)
+#     pl.enable_fly_to_right_click(callback=callback)
+#     pl.show(auto_close=False)
+#     width, height = pl.window_size
+#     cpos_before = pl.camera_position
+#     pl.iren._mouse_right_button_press(width//4, height//2)
+
+#     # ensure callback was called and camera position changes due to "fly"
+#     assert cpos_before != pl.camera_position
+#     assert point
```

Comment:
It is good to test also in multi_render. Is there a reason to comment out?
```suggestion


def test_enable_fly_to_right_click_multi_render(sphere):
    """Same as enable as fly_to_right_click except with two renders for coverage"""
    point = []
    def callback(click_point):
        point.append(click_point)
    pl = pyvista.Plotter(shape=(1, 2))
    pl.add_mesh(sphere)
    pl.enable_fly_to_right_click(callback=callback)
    pl.show(auto_close=False)
    width, height = pl.window_size
    cpos_before = pl.camera_position
    pl.iren._mouse_right_button_press(width//4, height//2)
     # ensure callback was called and camera position changes due to "fly"
    assert cpos_before != pl.camera_position
    assert point
```

Reply:
I commented this out because I wasn't sure if including it was necessary since it doesn't improve coverage, but it's probably a good test to have regardless.